src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -46,6 +51,49 @@ def register_stratifications(self, builder: Builder) -> None:
         builder.results.register_stratification(
             "sex", ["Female", "Male"], requires_columns=["sex"]
         )
+        builder.results.register_stratification(
+            "child_sex",
+            ["Female", "Male", "invalid"],
+            ["invalid"],
+            requires_columns=[COLUMNS.SEX_OF_CHILD],
+        )
+        builder.results.register_stratification(
+            "child_age_group",
+            self.child_age_bins["age_group_name"].to_list(),
+            mapper=self.map_child_age_groups,
+            is_vectorized=True,
+            requires_columns=[COLUMNS.CHILD_AGE],
+        )
+
+    def get_child_age_bins(self, builder: Builder) -> pd.DataFrame:
```

Comment:
The next two methods are a copy paste of what is already happening but is for the neonatal age groups and we can't parse it from the config file since the config file is for the mothers and that is what the results stratifier uses for age start and age end (min and max) values